src/fitnesse/testsystems/slim/SlimTestContextImpl.java

```diff
@@ -14,6 +15,15 @@
   private final Map<String, String> symbols = new HashMap<String, String>();
   private final Map<String, ScenarioTable> scenarios = new HashMap<String, ScenarioTable>();
   private final TestSummary testSummary = new TestSummary();
+  private TestPage pageToTest;
+
+  public SlimTestContextImpl(TestPage pageToTest) {
+    this.pageToTest = pageToTest;
+  }
+  
+  public SlimTestContextImpl() {
+    super();
+  }
```

Comment:
I think deleting the default constructor and just have the unit tests pass in `null` is better (but more work, maybe your IDE can help you there).
Furthermore I would suggest making the field `pageToTest` final.